components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/resources/authorizedApps.yaml

```diff
@@ -233,6 +233,28 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  '/authorized-apps/{application-id}/tokens':
+    delete:
+      tags:
+        - admin
+      summary: "Removes all the tokens granted for an application"
+      operationId: deleteIssuedTokensByAppId
+      description: |
+        Removes all the tokens granted for a given app ID
```

Comment:
better include the permission level and scope if that is avaialble?

Reply:
Scope isn't available for this API.

Reply:
Addressed in [4a16edd](https://github.com/wso2/identity-api-user/pull/129/commits/4a16edd6a73570b5b171cd2b9b8ca73a578796d7)